configure

```diff
@@ -4989,6 +4989,10 @@ if test -n "$ac_unrecognized_opts"; then
   esac
 fi
 
+CFLAGS="$CFLAGS -Werror"
+CPPFLAGS="$CPPFLAGS -Werror"
+CXXFLAGS="$CXXFLAGS -Werror"
```

Comment:
i'd preferably like to make this possible to disable (like through a `--disable-werror` flag), but i'm not sure how to exclude this flag from `$ac_unrecognized_opts`.